model/riscv_sys_regs.sail

```diff
@@ -560,11 +607,11 @@ register mtime : bits(64)
 register minstret : bits(64)
 register minstret_written : bool
 
-function retire_instruction() -> unit = {
+function retire_instruction(orig_priv : Privilege) -> unit = {
```

Comment:
priv is fine, it's clearly referring to the instruction being retired

Reply:
I wasn't sure what you mean by this. Are you saying that this function can just look at the current privilege, therefore needs no argument? If you can be more specific I will try to incorporate your suggestion. Thanks very much!